resources/views/user/invite/create.blade.php

```diff
@@ -69,6 +69,18 @@ class="form__textarea"
                             {{ __('common.message') }}
                         </label>
                     </p>
+                    @if ($user->group->is_modo)
+                    <p class="form__group">
+                        <textarea
+                            id="internal_note"
+                            class="form__textarea"
+                            name="internal_note"
```

Comment:
You might need to add `placeholder=" "` here for the autoexpand to work in browsers that don't yet support `field-sizing: content` as well as to get the label to float.